src/classes/CMT_UnitTestData_TEST.cls

```diff
@@ -170,15 +173,33 @@ public class CMT_UnitTestData_TEST {
                         '"QualifiedApiName":"{3}"' +
                         '!~,';
                 String dummyId = '000000000000000AAA';
-                fldName = this.fldName.replace('__c', '__r');
                 fieldJSON += ',' + String.format(compountFieldTemplate, new List<String>{
-                        dummyId, this.fldName, this.relatedObject, (String)this.fldValue, entityType });
+                        dummyId, fixedFieldName, fixedObjectName, (String)this.fldValue, entityType });
             }
 
             return fieldJSON.removeEnd(',');
         }
     }
 
+    /**
+     * @description Prefixes the package namespace to custom fields/objects only if they do not already have
+     * a namespace. Needed because making a single call to StrAllNSPrefix() could not handle the many variations
```

Comment:
I had thought that `StrAllNSPrefix()` was used in many places by just passing in a long string rather than one field at a time, however doing another global search shows differently. This method is pretty much only used by UTIL_Namespace_TEST, the new rollups code, and one BDI class. I'll update the UTIL method accordingly.